cereal/car.capnp

```diff
@@ -301,8 +301,12 @@ struct CarParams {
   tireStiffnessRear @14 :Float32;    # [N/rad] rear tire coeff of stiff
 
   # Kp and Ki for the lateral control
-  steerKp @15 :Float32;
-  steerKi @16 :Float32;
+  steerKpBP @42 :List(Float32);
+  steerKpV @43 :List(Float32);
+  steerKiBP @44 :List(Float32);
+  steerKiV @45 :List(Float32);
+  steerKpDEPRECATED @15 :Float32;
```

Comment:
@vntarasov , since carParams isn't logged, you can remove the DEPRECATED ones.

Reply:
Thanks! Should I then clean up existing DEPRECATED carParams fields as well?

Reply:
No worries, I'll do the clean up on our internal branch, since it will generate a lot of index shifting.